src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -157,16 +157,15 @@ public enum OperationKind
 
         // Operations that are constituents of statements, expressions, or declarations.
 
+
+        // Unused 0x400 and 0x402
```

Comment:
Would prefer if this said something about those numbers being available for use.